[2022-11-08T00:19:34.270+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2022-11-08T00:19:29.059613+00:00 [queued]>
[2022-11-08T00:19:34.301+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2022-11-08T00:19:29.059613+00:00 [queued]>
[2022-11-08T00:19:34.303+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-08T00:19:34.305+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-08T00:19:34.306+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-08T00:19:34.344+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): inaccurate> on 2022-11-08 00:19:29.059613+00:00
[2022-11-08T00:19:34.352+0000] {standard_task_runner.py:55} INFO - Started process 3455 to run task
[2022-11-08T00:19:34.360+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'manual__2022-11-08T00:19:29.059613+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpe282759o']
[2022-11-08T00:19:34.361+0000] {standard_task_runner.py:83} INFO - Job 15: Subtask inaccurate
[2022-11-08T00:19:34.481+0000] {task_command.py:376} INFO - Running <TaskInstance: my_dag.inaccurate manual__2022-11-08T00:19:29.059613+00:00 [running]> on host 786e58bb7830
[2022-11-08T00:19:34.592+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=inaccurate
AIRFLOW_CTX_EXECUTION_DATE=2022-11-08T00:19:29.059613+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-08T00:19:29.059613+00:00
[2022-11-08T00:19:34.624+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-08T00:19:34.636+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "echo 'inaccurate'"]
[2022-11-08T00:19:34.662+0000] {subprocess.py:86} INFO - Output:
[2022-11-08T00:19:34.672+0000] {subprocess.py:93} INFO - inaccurate
[2022-11-08T00:19:34.674+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-08T00:19:34.715+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20221108T001929, start_date=20221108T001934, end_date=20221108T001934
[2022-11-08T00:19:34.776+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-08T00:19:34.811+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
